prompt-scrape/conversation.py

```diff
@@ -101,3 +103,30 @@ def conversation_collector(self,
                     logger.exception(f"Error processing prompt: {e}")
 
                 progress.update(1)
+
+if __name__ == "__main__":
+    # Parse command line arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input_file", help="Input file name")
+    parser.add_argument("output_file", help="Output file name")
+    parser.add_argument("-k", "--openai_api_key", help="OpenAI API key")
+    args = parser.parse_args()
+
+    # Get the OpenAI API key
+    openai_api_key = args.openai_api_key or os.environ.get("OPENAI_API_KEY")
```

Comment:
Is there support for multiple keys here? Like 20 api keys for example

Reply:
I believe that still works through using the environment variable?